src/reducers/productReducer.js

```diff
@@ -0,0 +1,35 @@
+export const ProductReducer = (state, action) => {
```

Comment:
use good naming convention for readability and understanding
- use productState instead of state
- use productAction instead of action

Reply:
Sure, will do it, thanks